force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -176,11 +214,12 @@ public with sharing class ClassesObjectsMethods {
      */
     public static String getCurrentDate() {
         // Get the todays date
-
+        Date today = date.today();
         // Format the current date
-
+        String locale = today.format();
+        System.debug(locale);
         // Return the result
-        return null; // Replace null with the variable you used to store the result
+        return locale; // Replace null with the variable you used to store the result
```

Comment:
Good work.  You can clean this up and just return the value directly. I  normally only assign to a variable if I then need to change that data in some way. 
  return Date.today().format();